src/services/cryptoService.js

```diff
@@ -0,0 +1,83 @@
+import * as jose from "jose";
+
+const idpApiURL = window._env_.IDP_API_URL;
+
+const privatePem = `-----BEGIN PRIVATE KEY-----
```

Comment:
why hardcode a private key?

Reply:
can u not pick from env?

Reply:
The private key moved into the env-config.js file. This file gets updated when docker is built based on the env variables.